src/chrome/index.js

```diff
@@ -4,17 +4,18 @@ import { setWorkerProfile } from '../actions';
 import listenMessages from './listenMessages';
 import handleWorkerStateNotifications from './handleWorkerStateNotifications';
 import handleWork from './handleWork';
+import getSync from './getSync';
 import handleStateSaving from './handleStateSaving';
 import renderIcon from './renderIcon';
 import startIdleChecking from './startIdleChecking';
 import handleCaptcha from './handleCaptcha';
 import { applyMiddleware } from 'redux';
 import { logMiddleware } from '../logging';
 
-export const startChromePlugin = (auth, pollUrl, chrome, socketConstructor = Socket) => {
+export const startChromePlugin = (chrome, socketConstructor = Socket) => {
   const reloader = () => window.location.reload(true);
   const enhancer = applyMiddleware(logMiddleware);
-  const plugin = startPlugin({ auth, pollUrl, enhancer, reloader, socketConstructor });
+  const plugin = startPlugin({ enhancer, reloader, socketConstructor });
```

Comment:
Do we need to leave the logic for handling `auth` and `pollUrl` in `startPlugin`?